scripts/populate-instance.sh

```diff
@@ -106,7 +106,7 @@ ${INVENIO_WEB_INSTANCE} db create
 
 # sphinxdoc-create-user-account-begin
 ${INVENIO_WEB_INSTANCE} users create \
-       --email ${INVENIO_USER_EMAIL} \
+       ${INVENIO_USER_EMAIL} \
```

Comment:
It's a widespread convention in Unix, see e.g. any system command manual page `man ls` and friends, the advertised usage synopses are `ls [options] ... <file> ...`, or `kill [options] <pid>`, etc... Kinda everywhere.